crescent/ext/tasks/loop.py

```diff
@@ -12,7 +12,7 @@
 class Loop(Task):
     def __init__(self, callback: TaskCallbackT, delay_seconds: float) -> None:
         self.delay_seconds = delay_seconds
-        self.first_loop = True
+        self.first_loop: bool = True
```

Comment:
I dont think you need to do this

Reply:
Pyright complains the type might not be concretely inferred. This is also running using `--verifytypes` which checks for typing coverage of public-facing objects, etc.

Reply:
tbf it can't hurt to have it really